### ISSUE TITLE
Running `kolibri services --foreground` results in TypeError

### ISSUE THREAD

**Issue by aronasorman** [learningequality/kolibri#8332]
## Observed behavior

Running `kolibri --services --foreground` results in this error, and not starting up:

```
Traceback (most recent call last):
  File "/usr/local/bin/kolibri", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.6/site-packages/kolibri/dist/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/kolibri/dist/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.6/site-packages/kolibri/dist/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.6/site-packages/kolibri/utils/cli.py", line 194, in invoke
    return super(KolibriDjangoCommand, self).invoke(ctx)
  File "/usr/local/lib/python3.6/site-packages/kolibri/dist/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.6/site-packages/kolibri/dist/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/kolibri/utils/cli.py", line 338, in services
    server.start(port=port, zip_port=None, serve_http=False, background=background)
  File "/usr/local/lib/python3.6/site-packages/kolibri/utils/server.py", line 589, in start
    zip_port = int(zip_port)
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```

This is running on Linux, inside a docker image.

**Comment by aronasorman** [comment]
It seems like we always pass in `None` when running services independently:

```python
def services(port, background):
    """
    Start the kolibri background services.
    """

    port = OPTIONS["Deployment"]["HTTP_PORT"] if port is None else port

    logger.info("Starting Kolibri background services")

    server.start(port=port, zip_port=None, serve_http=False, background=background)
```

**Comment by rtibbles** [comment]
Ah, sorry @aronasorman - we no longer use the `services` command during development, so this is probably getting a lot less day to day testing than it used to. Can fix.

**Comment by aronasorman** [comment]
No worries, happy to be a QA tester here 👓

**Comment by rtibbles** [comment]
Fixed in #8334
